src/policy_downloader.rs

```diff
@@ -66,6 +66,8 @@ impl Downloader {
         );
 
         let mut policy_verification_errors = vec![];
+        let verification_config_default = LatestVerificationConfig::default();
+        let verification_config = verification_config.unwrap_or_else(|| &verification_config_default);
```

Comment:
I would prefer to not create a default verification config. Right now verification will not pass if one uses VerificationConfig that contains 0 keys (correct behaviour), so this isn't an issue, but makes the code look weird.

I would prefer to use the `verification_config` as Option that we receive, and just skip the verification parts in this function (and called ones) if `verification_config` is `None`. This encapsulates the code and functionality, which simplifies things imho.